Fix incorrect image on Bottleneck Event Bubble

bmclaughlin opened the pull request:
https://bugzilla.redhat.com/show_bug.cgi?id=1280104

bmclaughlin commented:
@h-kataria please review.

@miq-bot add_label bug, ui

h-kataria commented:
@bmclaughlin can you add a spec test for this.

dclarizio commented:
@bmclaughlin please add a screen shot as well.

Fryguy commented on lib/report_formatter/timeline.rb:
```diff
@@ -87,7 +87,13 @@ def tl_event(tl_xml, row, col)
           #         e_title = "#{ui_lookup(:model=>rec[:resource_type])}: #{rec[:resource_name]}"
           e_title = rec[:resource_name]
-          e_image = "/images/icons/new/#{bubble_icon(rec[:resource_type])}.png"
+          e_image_name = if rec[:resource_type].eql?("ExtManagementSystem") &&
```
Prefer `==` over `eql?`

Fryguy replied:
Also, here, if rec is the object, then this just becomes `rec.resource.kind_of?(ExtManagementSystem)`

Fryguy commented on lib/report_formatter/timeline.rb:
```diff
@@ -87,7 +87,13 @@ def tl_event(tl_xml, row, col)
           e_title = rec[:resource_name]
-          e_image = "/images/icons/new/#{bubble_icon(rec[:resource_type])}.png"
+          e_image_name = if rec[:resource_type].eql?("ExtManagementSystem") &&
+                            ExtManagementSystem.find(rec[:resource_id]).emstype.eql?("rhevm")
```
Is rec a Hash here, or is it the actual record?, if it's the actual record, you would just do `rec.resource.ems_type == "rhevm"`

Fryguy commented on lib/report_formatter/timeline.rb:
```diff
@@ -87,7 +87,13 @@ def tl_event(tl_xml, row, col)
+                            ExtManagementSystem.find(rec[:resource_id]).emstype.eql?("rhevm")
+                           "vendor-redhat"
+                         else
+                           rec[:resource_type]
```
This feels wrong...resource_type is likely in SnakeCase::WithColons, so it couldn't be an image name

chessbyte commented:
@bmclaughlin Please rebase and push again

bmclaughlin commented:
Rebased and addressed feedback from @Fryguy.  I moved my changes to the existing bubble_icon method to encapsulate it better.  Screenshots and specs coming soon.

bmclaughlin commented:
![1280104_after](https://cloud.githubusercontent.com/assets/1630348/13049259/9acc74be-d3b9-11e5-810f-fd1ec12f3585.png)
![1280104_before](https://cloud.githubusercontent.com/assets/1630348/13049260/9acf6408-d3b9-11e5-8638-eeb312c64175.png)

chessbyte commented:
@dclarizio have the screenshot
@h-kataria waiting for spec

h-kataria commented:
@bmclaughlin are you working on spec test for this fix?

bmclaughlin commented:
@h-kataria, yes, should be finished today.

bmclaughlin commented:
@h-kataria specs added, thanks!

h-kataria commented:
:+1:  ready to merge when Travis is green.
